dev/langtool/meta/ckeditor.plugin-a11yhelp-dialogs/meta.txt

```diff
@@ -35,6 +35,10 @@ legend.1.items.8.name = Label for the Access Next Focus Space Command section of
 legend.1.items.8.legend = Keyboard shortcut instructions for the Access Next Focus Space command displayed in the Accessibility Instructions dialog window.
 legend.1.items.9.name = Label for the Accessibility Help section of the Accessibility Instructions dialog window.
 legend.1.items.9.legend = Keyboard shortcut instructions for the Accessibility Help command displayed in the Accessibility Instructions dialog window.
+legend.1.items.10.name = Label for the Paste as Plain Text Command section of the Accessibility Instructions dialog window.
+legend.1.items.10.legend = Keyboard shortcut instructions for the Paste as Plain Text command displayed in the Accessibility Instructions dialog window.
+legend.1.items.11.name = Label for the Paste as Plain Text, followed by Paste Command section of the Accessibility Instructions dialog window.
```

Comment:
Rather than adding edge label as a separate item, let's add it as a property in item related to Pasting as plain text.